JobConfig/common/geom_nofield.txt

```diff
@@ -1,13 +1,4 @@
 //
-//  '0-current' DS field configuration
-//
-#include "Production/JobConfig/common/geom_baseline.txt"
-vector<string> bfield.innerMaps = {
-  "BFieldMaps/Mau13/DS-nofield.txt",
-  "BFieldMaps/Mau13/PSMap.header",
-  "BFieldMaps/Mau13/TSuMap.header",
-  "BFieldMaps/Mau13/TSdMap.header",
-  "BFieldMaps/Mau13/PStoDumpAreaMap.header",
-  "BFieldMaps/Mau13/ProtonDumpAreaMap.header",
-  "BFieldMaps/Mau13/DSExtension.header"
-};
+#include "Offline/Mu2eG4/geom/geom_common.txt"
+#include "Offline/Mu2eG4/geom/bfgeom_no_field.txt"
```

Comment:
Once Offline PR #685 is merged, you will be able to replace the two lines with just
#include "Offline/Mu2eG4/geom/geom_common_no_field.txt"

Reply:
good suggestion, done.